docs/src/tutorials/tutorial_1.md

```diff
@@ -4,4 +4,336 @@
 
 If you are reading this tutorial, I am going to assume this is the first time you are using `Javis` to create an animation. In that case, welcome to `Javis`! 😃 By following this tutorial, we are going to make you a director of your very own animations written in pure Julia! 🎬 🎥
 
-If you have not installed `Javis` yet, please visit the homepage to [read the installation instructions](../index.html#Installation).
+If you have not installed `Javis` yet, please visit the homepage to [read the installation instructions](/index.html#Installation).
+
+## Explanation
+
+Luxor.jl drawing onto a canvas. It provides simple functions like line, circle and so on and does animation. If you're interested in 2D graphics, you should definitely check out the awesome Luxor Tutorial.
+
+In order to use `Javis.jl`, you will need to include the following in your file.
+
+```julia
+using Javis
+
+javis(
+    [[video]], # Video struct
+    [[actions]], # Vector
+    pathname=[[pathname]] # String
+)
+```
+
+`javis()` has three parameters: 
+* `video`
+* `actions`
+* `pathname`
+
+### pathname
+
+The `pathname` is a String. In order to get the final result of `javis`, you will need to set `pathname` to the path where the gif will be stored and the gif name. Similar to the following:
+
+```julia
+javis(
+    [[video]], # Video struct
+    [[actions]], # Vector
+    pathname="/path/animation.gif"
+)
+```
+
+### Video
+
+The `video` is a Video struct and its constructor arguments are width and height.
+
+```julia
+video = Video(500,500) # 500x500 // width x height
+javis(
+    video,
+    [[actions]], # Vector
+    pathname="/path/animation.gif"
+)
+```
+
+### Actions
+
+The `actions` is a Vector of actions. There are two types of actions:
+
+* `BackgroundAction`
+* `Action`
+
+```julia
+video = Video(500,500)
+javis(
+    video,
+    [
+        ...
+    ],
+    pathname="/path/animation.gif"
+)
+```
+
+#### BackgroundAction()
```

Comment:
Let me add some comments of the difference between `BackgroundAction` and `Action` here.
A BackgroundAction is basically the same as an Action but it leaks everything to the outside. What that means is:
If you have `Action([[frames]], ground)` and set `sethue("blue")` it will **not** be leaked to the outside word and only exists for this action. For `BackgroundAction` it is the global layer so it actually sets the default color to blue. Hope that makes sense